src/client/cypress/integration/Demo/trade.js

```diff
@@ -0,0 +1,32 @@
+import { Given, When, Then, And} from 'cypress-cucumber-preprocessor/steps'
+
+
+
+Given('Reactive Trader is open', () => {
+
+    //cy.visit("https://web.prod.reactivetrader.com/")
+    cy.visit("http://localhost:1920")
+
+
+
+})
+And('{string} price tile is displayed', (combination) => {
+    cy.VerifyCurrencyOnPage(combination)
+
+})
+
+When('user performs {string} trade for {string} of {string}', (operation, quantity, combination) => {
```

Comment:
To be consistent with the naming/types
operation -> direction
quantity -> notional
combination -> symbol

Reply:
fixing this comment in new commit